### PR TITLE
[Hs 269] navigat to expected screen after login modal

### PR DESCRIPTION
**Pull request by dlvinh**

### CODE REVIEW

**Review by hophongit98 — DISMISSED** [review]

**Review by hophongit98 — COMMENTED** [review]
pls ignore my approval, my mistake ✌️

File: lib/features/dashboard/dashboard_screen.dart [hunk 1]
```diff
@@ -172,13 +173,28 @@ class _DashboardBodyState extends State<DashboardBody> {
...
+              BottomBarItems item = context
+                  .read<DashboardInteractionCubit>()
+                  .pressedBottomBarItem;
+              // check prev state is close login modal
+              if (dashBoardState is CloseHsModal) {
+                context
+                    .read<DashboardInteractionCubit>()
+                    .onBottomItemTapped(item);
+              }
+            }
```
[the remark this answers is not in the document]
Comment (suesitran) [reply lib/features/dashboard/dashboard_screen.dart:190]: Agree, please move this logic into cubit

**Comment by DongNguyenQ** [comment]
@dlvinh LGTM

File: lib/features/dashboard/view_model/dashboard_interaction_cubit.dart [hunk 2]
```diff
@@ -106,6 +118,16 @@ class DashboardInteractionCubit extends Cubit<DashboardInteractionState> {
     }
   }
 
+  void navigateToExpectedScreen() {
+    if (state is CloseLoginModal) {
+      if (pressedBottomBarItem == BottomBarItems.addingProperty) {
+        // TODO: Adding Property navigate flow
```
Comment (hophongit98) [inline lib/features/dashboard/view_model/dashboard_interaction_cubit.dart:124]:
```suggestion
        onAddPropertyPressed();
```

File: lib/models/authentication/authentication_service.dart [hunk 3]
```diff
@@ -34,6 +37,10 @@ class AuthenticationService {
     });
   }
 
+  void closeUserDetialStream() {
```
Comment (hophongit98) [inline lib/models/authentication/authentication_service.dart:40]:
```suggestion
  void closeUserDetailStream() {
```

File: lib/models/authentication/authentication_service.dart [hunk 4]
```diff
@@ -13,14 +13,17 @@ class AuthenticationService {
...
     GoogleSignIn? googleSignIn,
     FirebaseAuth? firebaseAuth,
     FacebookAuth? facebookAuth,
   })  : _googleSignIn = googleSignIn ?? GoogleSignIn(),
         _firebaseAuth = firebaseAuth ?? FirebaseAuth.instance,
-        _facebookAuth = facebookAuth ?? FacebookAuth.instance {
-    _firebaseAuth.authStateChanges().listen((event) {
+        _facebookAuth = facebookAuth ?? FacebookAuth.instance;
+
+  void initialiseUserDetialStream() {
```
Comment (hophongit98) [inline lib/models/authentication/authentication_service.dart:25]:
```suggestion
  void initialiseUserDetailStream() {
```

**Review by suesitran — APPROVED** [review]

**Pull request merged**